data/mods/gen3colosseum/scripts.ts

```diff
@@ -0,0 +1,21 @@
+export const Scripts: ModdedBattleScriptsData = {
+	inherit: 'gen3',
+	gen: 3,
+
+	checkWin(faintData?: Battle['faintQueue'][0]) {
+		const team1PokemonLeft = this.sides[0].pokemonLeft + (this.sides[0].allySide?.pokemonLeft || 0);
+		const team2PokemonLeft = this.sides[1].pokemonLeft + (this.sides[1].allySide?.pokemonLeft || 0);
+		const team3PokemonLeft = this.gameType === 'freeforall' && this.sides[2]!.pokemonLeft;
+		const team4PokemonLeft = this.gameType === 'freeforall' && this.sides[3]!.pokemonLeft;
+		if (!team1PokemonLeft && !team2PokemonLeft && !team3PokemonLeft && !team4PokemonLeft) {
```

Comment:
```suggestion
		if (this.sides.every(side => !side.pokemonLeft)) {
```

Reply:
Should I roll this change to the sim/battle.ts as well?